doc/api/fs.md

```diff
@@ -1549,6 +1549,23 @@ Synchronous version of [`fs.readFile`][]. Returns the contents of the `file`.
 If the `encoding` option is specified then this function returns a
 string. Otherwise it returns a buffer.
 
+_Note: The behavior of `fs.readFileSync()` is platform specific. As such,
+reading a directory on macOS, Linux and Windows will return an error.
+On FreeBSD, the contents of the files in directory will be returned.
```

Comment:
I've been working (slowly) on getting as much consistency as possible in the `Note` sections in the docs... Can you please make this...

```
*Note*: The behavior...
```

... and do not bold the entire paragraph.